composer.json

```diff
@@ -23,7 +23,9 @@
             "composer phpcs:setup",
             "phpcbf -p"
         ],
-        "phpstan": "phpstan"
+        "phpstan": "phpstan",
+        "phpunit": "phpunit",
+        "phpunit:coverage": "phpunit --coverage-text --coverage-html public/coverage"
```

Comment:
At the moment that will only work if you have a local php installation with some existent coverage provider like `pcov` configured and enabled (like in the `.gitlab-ci.yml`) which is not the case in the workspace container at the moment.

Reply:
:upside_down_face:  https://github.com/engelsystem/engelsystem/blob/main/docker/dev/Dockerfile#L5-L7

Reply:
At least it was not *looksconfused

I mean: i explicitly tested it...